sdk/src/index.ts

```diff
@@ -1,42 +1,108 @@
 import { Keyring } from '@polkadot/api'
+import { getBalance, transferBalance } from '@scio-labs/use-inkathon'
 import dotenv from 'dotenv'
 import { Invariant } from './invariant.js'
 import { Network } from './network.js'
-import { getDeploymentData, getEnvAccount, initPolkadotApi, printBalance } from './utils.js'
+import { PSP22 } from './psp22.js'
+import {
+  getDeploymentData,
+  getEnvAccount,
+  getEnvTestAccount,
+  initPolkadotApi,
+  printBalance
+} from './utils.js'
+import { WrappedAZERO } from './wrapped_azero.js'
 dotenv.config()
 
 const main = async () => {
   const network = Network.getFromEnv()
-  console.log(`Using ${network}`)
+  console.log(`using ${network}`)
+
   const api = await initPolkadotApi(network)
+
   const keyring = new Keyring({ type: 'sr25519' })
   const account = await getEnvAccount(keyring)
+  const testAccount = await getEnvTestAccount(keyring)
+
   await printBalance(api, account)
+  await printBalance(api, testAccount)
 
-  const { abi, wasm } = await getDeploymentData()
-  const invariant = new Invariant(api, account, network)
+  // deploy invariant
+  const invariantData = await getDeploymentData('contract')
+  const invariant = new Invariant(api, network)
 
   const initFee = { v: 10 }
-  const deployContract = await invariant.deploy(abi, wasm, initFee)
-  await invariant.load(deployContract.address, abi)
+  const invariantDeploy = await invariant.deploy(
+    account,
+    invariantData.abi,
+    invariantData.wasm,
+    initFee
+  )
+  await invariant.load(invariantDeploy.address, invariantData.abi)
+
+  // deploy token
+  const tokenData = await getDeploymentData('psp22')
+  const token = new PSP22(api, network)
 
-  const initialFee = await invariant.getProtocolFee()
-  console.log(initialFee)
+  const name = api.createType('Option<String>', 'Coin')
+  const symbol = api.createType('Option<String>', 'COIN')
 
+  const tokenDeploy = await token.deploy(
+    account,
+    tokenData.abi,
+    tokenData.wasm,
+    1000,
+    name,
+    symbol,
+    0
+  )
+  await token.load(tokenDeploy.address, tokenData.abi)
+
+  // deploy wrapped azero
+  const wazeroData = await getDeploymentData('wrapped_azero')
+  const wazero = new WrappedAZERO(api, network)
+
+  const wazeroDeploy = await wazero.deploy(account, wazeroData.abi, wazeroData.wasm)
```

Comment:
It's worth using the testnet contract when it exists in the testnet. This emulates the way we will use the SDK on the mainnet or testnet. So, it would be better to just load it on the testnet, instead of deploy and load